pulp_ansible/app/management/commands/add-sigstore-verifying-service.py

```diff
@@ -26,7 +26,7 @@
 
 DEFAULTS = {
     "rekor-url": "https://rekor.sigstore.dev",
-    "tuf-url": "https://sigstore-tuf-root.storage.googleapis.com/",
+    "tuf-url": "https://tuf-repo-cdn.sigstore.dev/",
```

Comment:
Is this going to be the final url?

Reply:
Yes, this is the one currently used for the public Sigstore instance